app/views/events/_form.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= form_with(model: @evento, local: true, class:"form-wrapper") do |form| %>
+  <% if @evento.errors.any? %>
```

Comment:
It is not necessary to use "@evento", you are already passing this variable when rendering the form (evento: @evento). Just use it as "evento".